source/_integrations/alexa.intent.markdown

```diff
@@ -395,6 +395,16 @@ text: !include alexa_confirm.yaml
 
 Alexa will now respond with a random phrase each time. You can use the include for as many different intents as you like so you only need to create the list once.
 
+## Workaround for having to say the skill's name
+
+Sometimes, you want to run script or scene intents without using the skill's name. For example, `Alexa <some script>` instead of `Alexa ask Home Assistant to run <some script>` because it is shorter.
```

Comment:
```suggestion
Sometimes, you want to run script or scene intents without using the skill's name. For example, 'Alexa <some script>' instead of 'Alexa ask Home Assistant to run <some script>' because it is shorter.
```